Moli/Core/Screens/Home/Live/MoliRoomItemView.swift

```diff
@@ -0,0 +1,111 @@
+import SwiftUI
+
+struct MoliRoomItemView: View {
+    var room: API.Room.HomeRoomResponse
+    
+    @State private var hosts: [API.Room.HostInfo] = []
+    @State private var viewerProfiles: [API.User.UserMinified] = []
+    @State private var extraViewersCount = 0
+    
+    private func setHostProfiles() {
+        self.hosts = room.hostsInfo
+    }
+    
+    private func setViewerProfiles() {
+        let viewerProfiles = Array(room.previewViewers.prefix(4))
+        
+        self.viewerProfiles = API.User.UserMinified.convertFrom(viewerProfiles)
+        self.extraViewersCount = self.room.viewers.count - viewerProfiles.count
+    }
+    
+    var body: some View {
+        ZStack(alignment: .topLeading) {
+            if let imageURL = room.thumbnail?.thumbnailUrl, let url = URL(string: imageURL) {
+                AsyncImage(url: url) { image in
+                    image.resizable()
+                        .ignoresSafeArea()
+                } placeholder: {
+                    Image("moliRoomsFallbackBG")
+                        .resizable()
+                        .ignoresSafeArea()
+                }
```

Comment:
❓ Finns det någon anledning till att ha dessa modifier:s? Spontant känns det som att de inte gör någon skillnad?

```suggestion
                AsyncImage(url: url) { image in
                    image.resizable()
                } placeholder: {
                    Image("moliRoomsFallbackBG")
                        .resizable()
                }
```